src/test/java/org/folio/dao/DataImportQueueDaoTest.java

```diff
@@ -37,35 +41,54 @@ public class DataImportQueueDaoTest  {
 
   @InjectMocks
   DataImportQueueItemDao queueItemDaoImpl = new DataImportQueueItemDaoImpl();
-  
 
   UUID storedItemUUID;
+
   @Before
   public void setUp() {
-    MockitoAnnotations.initMocks(this);
+    MockitoAnnotations.openMocks(this);
     when(postgresClientFactory.getInstance()).thenReturn(postgresClient);
   }
-  @Test 
+
+  @Test
   public void shouldAddQueueItem(TestContext context) {
     // given
-    when(postgresClient.execute(anyString(), any(Tuple.class)))
-      .thenReturn(Future.succeededFuture(new LocalRowSet(1)));
+    doAnswer((InvocationOnMock invocation) -> {
```

Comment:
This was the main change.  When mocking `void` functions, the normal `when(mock.method(parameters)).thenAnswer/thenReturn/etc` does not work; instead, it has to be flipped to be `doAnswer(...).when(mock).method(parameters)` due to some technicalities with the way `void` works.